src/CloudFtpBridge.Infrastructure.FTP/FTPFileSystem.cs

```diff
@@ -0,0 +1,71 @@
+﻿using CloudFtpBridge.Core.Services;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CloudFtpBridge.Infrastructure.FTP
+{
+    public class FTPFileSystem : IFtpSystem
```

Comment:
This class should implement `IFileSystem`. Most of the code you have in Send.cs and Receive.cs can then be ported here.